### DISCUSSION TITLE
Is DualStack on the roadmap?

### DISCUSSION THREAD

**Discussion by OrvilleQ** [tonarino/innernet#235] [Q&A]
Hello there~

I'm trying to use innernet to create my kubernetes edge-cluster, then I found that I can't create an interface with both ipv4 and ipv6 enabled, the `innernet-server` said `your network CIDR should be in the RFC1918 IPv4 (10/8, 172.16/12, or 192.168/16), or RFC4193 IPv6 (fd00::/8) ranges.`.

So is dualstack on the roadmap?

**Comment by mcginty** [comment]
It's not a high priority right now, just because there's a workaround of running two innernet networks (so two interfaces), which I've done in the past with success.

You might also be able to define your innernet CIDR as an IPv4-mapped-IPv6 range, and possibly trick IPv4 applications into running successfully on that stack, but I haven't experimented with that myself.
